apps/bsd/src/contexts/AppContext.ts

```diff
@@ -1,16 +1,21 @@
+import { Election } from '@votingworks/types'
 import { createContext } from 'react'
 import { MachineConfig } from '../config/types'
 
 interface AppContextInterface {
   usbDriveStatus: string
   usbDriveEject: () => void
   machineConfig: MachineConfig
+  election: Election | undefined
+  electionHash: string | undefined
```

Comment:
```suggestion
  election?: Election
  electionHash?: string
```

Or this to ensure they're both there or both absent?

```suggestion
  electionDefinition?: ElectionDefinition
```

Reply:
Yeah I didn't make it electionDefinition because the way AppRoot is set up it can have election defined but not electionHash. That should be fixed and this should be switched to electionDefinition but I didn't want to wrap all of that into this change.